web/crux-ui/README.md

```diff
@@ -7,4 +7,11 @@ Server-side TLS is in use, the server is the `crux` component, the server's publ
 ### Install `npm` modules in Apple Silicon
 
 You have to explicit add the target architecture:
```

Comment:
Even if it is not yours, could you please fix the sentence above? eg. explicit -> explicitly